### ISSUE TITLE
Show new margin call price for partial margin cover

### ISSUE THREAD

**Issue by drltc** [BitShares/web_wallet#298]
When covering a short, there are generally two possible scenarios:
- (1) I want to fully exhaust the short, exiting the position entirely and liberating the collateral.
- (2) I want to maintain the short exposure, but raise the margin call price.

Situation (1) is adequately covered by the current UI.  Situation (2) however, is not.

For example, I'm short $100 BitUSD with a margin call price of $30 / BTSX -- how much will that margin call price increase if I cover $10?  The UI should put that information at my fingertips.

That functionality would certainly be useful, but I'd be even more interested in how many BitUSD I need to cover to raise the margin call price to $40 / BTSX.  So I should be able to type in 40 as my desired margin call price.  Then the UI should update the number of USD to cover accordingly.

In other words, the "number of USD to cover" and "new margin call price" fields should both be editable, and typing in either one should cause the other to update accordingly.
